docs/SERVICE_CONTINUITY.md

```diff
@@ -0,0 +1,147 @@
+# Service Continuity for Make an LPA
+
+A service for members of the public to make a Lasting Power of Attorney. Users fill out online forms, pay a fee and can download the completed forms as a PDF which they can then sign and post to the Office of the Public Guardian to be registered.
+
+When the user creates an LPA in Make an LPA, details of the LPA are only stored in the Make an LPA Postgres database. These details are not entered into Sirius until the printed form is received by OPG and the LPA is entered into Sirius by a caseworker.
+
+The service is developed and operated by OPG Digital; see [the overview in our technical docs](https://docs.opg.service.justice.gov.uk/documentation/support/lpa.html)
+
+
+## Service Accountability (Accountable / Responsible)
+
+A product team within OPG Digital owns the delivery and maintenance of the service. The multi-disciplinary team is responsible for the service. We follow an agile software development lifecycle as mandated by the GDS Service Standard. Make an LPA Online has passed its Live assessment stage as part of the Service Standard.
+
+OPG Digital uses an embedded WebOps, which means infrastructure engineers are embedded within the team to support the product's pipelines and infrastructure, as well as developers who maintain the application code. OPG Digital follows a continuous delivery model, where we constantly deploy small chunks of change as soon as they are merged into the codebase. Comprehensive test automation ensures this is possible.
+
+### Contacts
+- [Service contact details](https://docs.opg.service.justice.gov.uk/documentation/support/lpa.html#service-team-contact)
+
+### Notes
+- The service will be replaced when the Modernised Digital LPA becomes publicly available.
+- Since October 2025 a seperate squad has been created to address some technical debt and ensure product support until decommissioned.
+
+## Service Scope
+
+The product covers the web application available at [https://www.lastingpowerofattorney.service.gov.uk/home](https://www.lastingpowerofattorney.service.gov.uk/home) for making an LPA document.
```

Comment:
could we refer to the root rather than home. We're not meant to link directly to the site and instead guide people in via GDS. I think providing the root address (that redirects to the GDS start page) still accurately describes the scope of the service.